web/src/routes/SurveyPages/SurveyFormPage.tsx

```diff
@@ -1,5 +1,400 @@
+  color: ${(props) => props.theme.colors.default};
+  background-color: ${(props) => props.theme.colors.container};
+  cursor: text;
+`;
+
+const QuestionContainer = styled(ItemContainer)``;
+
+const AnswerLable = styled.label`
+  font-size: 13px;
+  padding: 1.2vh 1.5vw 1.2vh 1.5vw;
+  color: ${(props) => props.theme.colors.text};
+  text-decoration: underline;
+  text-decoration-style: dotted;
+  text-decoration-color: currentColor;
+  border: ${(props) => props.theme.border};
+  border-radius: ${(props) => props.theme.borderRadius};
+`;
+
+const OptionContainer = styled.div``;
+
+const OptionInput = styled(TextInput).attrs({ type: 'text' })`
+  width: 30vw;
+  font-size: 13px;
+  margin-top: 3px;
+`;
+
+const DeleteOptionButton = styled.button`
+  font-weight: 900;
+  text-align: center;
+  color: ${(props) => props.theme.colors.text};
+  margin-left: 3px;
+  padding: 10px;
+  background-color: ${(props) => props.theme.colors.button};
+  width: 35px;
+  height: 35px;
+  border-radius: 50%;
+  border: none;
+  cursor: pointer;
+
+  &:hover {
+    opacity: 0.8;
+  }
+`;
+
+const SubmitButton = styled.button.attrs({ type: 'submit' })`
+  width: 15vw;
+  border: none;
+  padding: 2vh 2vw 2vh 2vw;
+  margin-top: 30px;
+  margin-bottom: 30px;
+  margin-left: 69vw;
+  border-radius: ${(props) => props.theme.borderRadius};
+  font-size: 2vh;
+  font-weight: 700;
+  color: ${(props) => props.theme.colors.default};
+  background-color: ${(props) => props.theme.colors.primary};
+  cursor: pointer;
+
+  &:hover {
+    background-color: ${(props) => props.theme.colors.prhover};
+  }
+`;
+
+// TODO: Drag and drop questions order
 export default function SurveyFormPage() {
-  return <div>SurveyFormPage</div>;
+  const [theme, toggleTheme] = useTheme();
+  const questionRefs = useRef<HTMLDivElement[]>([]);
+  const [recentCreate, setRecentCreate] = useState<number>();
+  const [resultModalOpen, setResultModalOpen] = useState<boolean>(false);
+  const [questionList, setQuestionList] = useState<QuestionCreateRequest[]>([]);
+  const [requiredCertificationList, setRequiredCertificationList] = useState<CertificationType[]>([]);
+  const [certificationIsChecked, setCertificationIsChecked] = useState<boolean>(false);
+  const [surveyData, setSurveyData] = useState<SurveyCreateRequest>({
+    title: '제목 없는 설문',
+    description: '설문지 설명',
+    startedDate: '2023-04-04T20:31',
+    endedDate: '2023-04-04T20:31',
+    certificationTypes: [],
+    questions: [],
+  });
+
+  const scrollToRecentCreateQuestion = () => {
+    if (typeof recentCreate !== 'undefined') {
+      questionRefs.current[recentCreate].scrollIntoView({ block: 'center', inline: 'nearest', behavior: 'smooth' });
+    }
+  };
+
+  useEffect(() => {
+    scrollToRecentCreateQuestion();
+  }, [recentCreate]);
+
+  useEffect(() => {
+    if (typeof surveyData !== 'undefined') {
+      setSurveyData({
+        ...surveyData,
+        questions: questionList,
+      });
+    }
+  }, [questionList]);
```

Comment:
This kind of useEffect will be bad for performance. You **should never set `Object` as dependency**.

If dependency is kind of `Object` which in our case `questionList`, this `SurveyFormPage` will be re-rendered on every time it is updated (even when it's been changed with the **SAME** value.) (See [https://react.dev/reference/react/useEffect#removing-unnecessary-object-dependencies](https://react.dev/reference/react/useEffect#removing-unnecessary-object-dependencies))

And super luckily, on JavaScript,

```javascript
const obj = {};
console.log(Object.is(obj, {}));
// Expected output: false
```
Thanks for using JavaScript :)

<img src="https://user-images.githubusercontent.com/63793178/230322539-59eae4aa-f2db-4c52-81c0-6520c9f9a204.png" width="300" />